mandatory/1-fix-functions.js

```diff
@@ -10,8 +10,8 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(isHappy) {
+  // let isHappy = true;
 
   if (isHappy) {
     return "I am happy";
```

Comment:
You forgot Else I am not happy

Reply:
It's there, I didn't have to edit that code. It was already right.